public/locales/en/topics/ancestor-veneration.json

```diff
@@ -33,5 +33,27 @@
     },
     "topicNav": {
         "labels": ["About this Topic", "Prayer Points", "Downloadables"]
-    }
+    },
+    "photos": [
+        { "src": "/images/0.png", "title": "scrambled it to make a type specimen book" },
```

Comment:
just a thought - will we need to have 'title' work with basic html tags? I'm not sure how we currently do it here will support <i> / <b> etc?

Reply:
Yeah I'm kind of assuming that these "captions" will be basic text, but this might be a concern if not